### PR TITLE
Added powered-by page

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Add a page with companies using or contributing to Pulsar

### CODE REVIEW

File: site2/website/data/users.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        url: 'https://www.sticorp.com.br/',
+        logo: 'https://www.sticorp.com.br/wp-content/uploads/2015/09/logo-horizontal.png',
+    },
+    {
+        name: 'Verizon Media',
+        url: 'https://www.verizonmedia.com/',
+        logo: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/09/Verizon_Media_2019_logo.svg/1200px-Verizon_Media_2019_logo.svg.png'
+    },
+    {
+        name: 'Yahoo! Japan',
```
Comment (nkurihar) [inline site2/website/data/users.js]: Could you modify to Yahoo! **JAPAN** ?
(It's YJ's formal service name)

Comment (merlimat) [reply site2/website/data/users.js]: Sure, fixed

Comment (nkurihar) [reply site2/website/data/users.js]: Thanks!

**Comment by cdbartholomew** [comment]
Would you mind adding Kafkaesque to the list?

`{
        name: 'Kafkaesque',
        url: 'https://kafkaesque.io/',
        logo: 'https://kafkaesque.io/wp-content/uploads/2019/04/color_logo_with_background.svg'
    },`

**Comment by jiazhai** [comment]
lgtm. will add some other companies that not included after this is merged.

**Comment by sijie** [comment]
run integration tests
run java8 tests

**Comment by merlimat** [comment]
run cpp tests

**Comment by merlimat** [comment]
run integration tests

**Comment by merlimat** [comment]
run integration tests

**Pull request merged**